content/tutorials/esc-setup-psp-creds/index.md

```diff
@@ -0,0 +1,128 @@
+
+```bash
+$ pulumi login
+
+Manage your Pulumi stacks by logging in.
+Run `esc login --help` for alternative login options.
+Enter your access token from https://app.pulumi.com/account/tokens
+    or hit <ENTER> to log in using your browser
+```
+
+## Obtain a personal access token
+
+Next, you'll also need a Pulumi Cloud [personal access token](/docs/pulumi-cloud/access-management/access-tokens/#personal-access-tokens). Create a short-lived access token for this tutorial, copy it into your current shell to login into Pulumi Cloud. Keep the token creation page open in a separate tab, or paste it somewhere temporarily to use in the next step of the tutorial.
+
+## Brief Explanation
+
+Pulumi Service Provider is used to create Pulumi Cloud resources using Pulumi IaC Programs. Just like an S3 bucket is created using an AWS provider, PSP allows creation of things like Environments, Environment Version Tags, Environment Webhooks, and much more. In order to authenticate with Pulumi Cloud, PSP uses the same personal access token (PAT for short!) as Pulumi CLI. This is where ESC comes in!
+
+Instead of providing PAT as an environment variable, risking losing or leaking it, we will create an ESC Environment and store the PAT there. We will then import this environment into the configuration of a new stack, in which we will create Pulumi Cloud resources
+
+## Create a new ESC project and environment
+
+In ESC, configuration settings belong to collections called [_environments_](/docs/esc/environments/working-with-environments/), and environments in turn belong to _projects_. Projects and environments may be created either with the Pulumi Cloud console or with the CLI.
+
+To get started, navigate to the Environments page using Pulumi Cloud console's navigation bar on the left. Then click `Create Environment` button, fill in `creds` for project name and `psp` for environment name and click `Create`. You will now see a blank new environment with some default commented-out explanations.
+
+## Fill in the PAT
+
+Select everything in the Environment definition pane and replace it with this:
+
+```bash
+values:
+  pulumiConfig:
+    "pulumiservice:apiUrl": % FILL IN CUSTOM BACKEND API HERE %
+    "pulumiservice:accessToken":
+      fn::secret: % FILL IN PAT HERE %
+```
+
+If you are *not* using Pulumi Self-hosted, just remove the line containing `apiUrl`. This optional field configures custom Pulumi Cloud backend, and is not needed for most users.
+
+Once you fill in your previously created PAT, click `Save` on the top right. This will update your new environment's definition with the values you just added. Note how the plain text PAT value you pasted in was converted into a secret. It is now securely stored within Pulumi Cloud, removing the risk of losing or leaking the PAT, while your stacks are still able to decrypt and use it. Feel free to close the token creation page now, we won't need it again!
```

Comment:
`Note that the plaintext PAT was converted into a secret value.`
Probably don't need the last sentence if we changed the line above when obtaining a PAT